modules/schema/src/main/java/org/apache/ignite/internal/schema/NativeTypes.java

```diff
@@ -195,6 +196,7 @@ public static NativeType timestamp() {
      * @param val Object to map to native type.
      * @return {@code null} for {@code null} value. Otherwise returns NativeType according to the value's type.
      */
+    @Contract("null -> null")
```

Comment:
What reason to add jetbrains annotations? I think we should start discussion at least internally before starting it.

Reply:
We already use jetbrains annotations in the project (see \@Nullable and \@NotNull).
The contract is clearly described in javadoc and the annotation just helps to suppress IDEA warning and affect nothing.